src/actors/containerization/services/postgresql/postgresql-scan/scan.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+
+"""
+Actor for extracting PostgreSQL related data from augeas output. This is useful for offline
+detection of PostgreSQL installation. 
+
+Output structure:
+
+"postgresql_scanner": {
+    "instances": [
+        {
+            "postgresql_conf": {
+                "path": String,
+                "properties": [
+                    {
+                        "name": string,
+                        "value": string
+                    },
+                    ...
+                ]
+            },
+            "pg_hba_conf": {
+                "path": String,
+                "properties": [
+                    {
+                        "name": string,
+                        "properties": [
+                            {
+                                "name": string,
+                                "value": string
+                            }
+                        ],
+                        ...
+                    },
+                    ...
+                ]
+            }
+        },
+        ...
+    ]
+}
+
+Improvements:
+  Currently the actor can detect only postgresql installed in default location reported by augeas.
+It would be nice, if it cold detect also non-standard installations used by i.e. Satellite 5 and
+probably few other RH products.
+"""
+
+import sys
+import json
+
+
+if __name__ == "__main__":
+    augeas_input = json.load(sys.stdin)
+
+    instances = [] 
+
+    # aug_postgresql might be undefined, that usually means the postgresql data
+    # were not initialized.
+    try:
```

Comment:
I think this exception is too wide. 

Origin of the failure it's not obvious, we don't know if we couldn't find ```aug_postgresql``` which means postgresql data weren't initialized, or data were initialized but we couldn't find some specific key which means something... 

Tomorrow we have demo, so I suggest to improve this in the next iteration, although right now i recommend to send message from KeyError exception to stderr (I mean original message from KeyError + short explanation what happened)